js-api-spec/importer.test.ts

```diff
@@ -729,6 +737,37 @@ it('throws an ArgumentError when the result sourceMapUrl is missing a scheme', (
   });
 });
 
+fit('compiles multiple nested relative imports and incorrect load path with renderSync', () => {
+  sandbox(dir => {
+    dir.write({
+      'src/index.scss': `
+        @import "./foo/_a.scss";
+        @import "./_includes.scss";
+      `,
+      'include/foo/_a.scss': '/* A */',
+      'include/foo/_b.scss': '/* B */',
+      'src/_includes.scss': `
+        @import "./foo/_a.scss";
+        @import "./foo/_b.scss";
+      `,
+    });
+
+    const result = renderSync({
```

Comment:
Legacy API test should be put `js-api-spec/legacy/importer.test.ts` instead.

Reply:
no wonder I didn't see other `renderSync` heh